.github/workflows/ci.yml

```
@@ -0,0 +1,135 @@
+name: Test + Docker
+
+on:
+  push:
```

```suggestion
  push:
    branches:
      - main
```

Should we restrict on push a bit so it doesn't double trigger jobs on pull requests (which may have another push)

---

done!